test/routes.inscription.test.js

```diff
@@ -151,6 +151,35 @@ describe("routes : inscription", () => {
           ti_ids.should.deep.equal([1, 2]);
         }));
 
+    it("should add user tis", () =>
+      chai
+        .request(server)
+        .post("/api/v1/inscription/mandataires")
+        .send({
+          username: "user_ti",
+          email: "test@test.com",
+          type: "ti",
+          pass1: "kikoo",
+          pass2: "kikoo",
+          cabinet: "2A",
+          tis: [1]
+        })
+        .then((res, req) => {
+          res.status.should.eql(200);
+        })
+        .then(async (res, req) => {
+          const user = await knex
```

Comment:
Mix of async/await and `.then` function call back. 
You should make the whole `it` `async` :heart: